docs/boot.md

```diff
@@ -38,18 +38,21 @@ import storage
 import usb_cdc
 import usb_hid
 
-# This is from the base kmk boot.py
-supervisor.set_next_stack_limit(4096 + 4096)
```

Comment:
Are we intentionally ending stack size overrides? Do boards generally have enough stack now without these hacks?

Reply:
Uhhh, we have boot.py still, but no boards that I have tested lately have required that, and have taken a hit on memory by doing so. I can add this back if it's desired. I left the stock file there for users to reference as well, and I may want to make a note specifically about that stack limit as opposed to just "forcing it" by leaving it in examples. No rp2040 board, even stacked with features, has required it.